pkgs/development/python-modules/pysvn/default.nix

```diff
@@ -63,8 +63,6 @@ buildPythonPackage rec {
     runHook postCheck
   '';
 
-  pythonImportCheck = [ "pysvn" ];
```

Comment:
I'm not sure how removing it is considered a fix. Can you explain?

Reply:
If I mark it broken rabbitvcs does not build anymore and the package is very custom and I think our importsCheck simply does not work with the package.

Reply:
I don't follow. If it declares a module that should be importable like any other module, no?

Reply:
RabbitVCS also does `import pysvn`, so I'm not sure why it would work for them, when it doesn't work for us.

Reply:
importing it in a python interpreter works but ``pythonImportCheck`` does not. I have no idea why but I guess that the package is not packaged in a standard way.